databrowser/src/domain/datasets/ui/tableView/toolBox/SearchAndFilterToolBoxPanel.vue

```diff
@@ -54,7 +54,18 @@ SPDX-License-Identifier: AGPL-3.0-or-later
           @filter="updateFilter(index, filter.operator, $event, true)"
         />
       </ToolBoxCardBody>
-      <ToolBoxCardBody>
+      <ToolBoxCardBody class="flex justify-between">
+        <ButtonCustom
+          class="flex items-center gap-2 p-2 py-1"
+          :size="Size.xs"
+          :variant="Variant.solid"
+          @confirmed-value="emit('filter', $event)"
+          @update:model-value="emit('update:modelValue', $event)"
```

Comment:
What's the reason to emit a `update:modelValue` here? The `ButtonCustom` does not emit such an event, so this binding has no effect. In addition, I'm not sure what we gain from a `update:modelValue`  event on a button.

If it's not needed, I'd remove the binding and `emit` definition on line 125